projects/oniguruma/.gitignore

```diff
@@ -0,0 +1,3 @@
+*~
```

Comment:
I dont this file is needed for OSS-Fuzz

Reply:
I assume they added because we don't have it in the main .gitignore though we probably should since backup files commonly use this suffix.

Reply:
https://github.com/google/oss-fuzz/pull/3736